src/main/java/com/allthemods/gravitas2/material/materials/GregitasFirstDegreeMaterials.java

```diff
@@ -81,5 +81,14 @@ public static void init() {
                         .components(Certus, 1, Hydrogen, 4)
         );
 
+        WroughtIronMagnetic = GregitasMaterials.registerMaterial(
+                new Material.Builder("wrought_iron_magnetic")
```

Comment:
Either set a custom lang or make the id `magnetic_wrought_iron`. Because currently this would be named "Wrought Iron Magnetic Ingot"

Reply:
id has been changed